### ISSUE TITLE
[BUG] - Linux Client 3.2.2 Flight Will Not Start

### ISSUE THREAD

**Issue by Rastuasi** [invernyx/smartcars-3-bugs#385]
### Describe the bug

Current install, been working for months, loaded up, new plugins installed for Smartcars3. Now when I click "Fly this Flight" it will not start, no response, no popup, no change in window. The error log shows a bad request.

### How do you reproduce this bug?

1. Load SmartCars3
2. Load Sim
3. Load My Flights Page
4. Hit Fly this Flight

Nothing happens.

### Expected behavior

Flight should start or at least a "cannot connect" message.

### Operating system

Manjaro Arch Linux

### Community airline

Walker Air

### smartCARS Version

Latest

### Plugins installed

chat, flight center, flight tracking, logbook, map

### Additional context

[bug_report_combined-2024-07-12.log](https://github.com/user-attachments/files/16199270/bug_report_combined-2024-07-12.log)

**Comment by Rastuasi** [comment]
Update: Deleted all configuration folders, client folders, downloaded a fresh version and allowed it to update. It seems the client updates did not successfully update all the port info, as I noticed they now differ. Was able to start a flight, but at least wanted to record the guess for reason for others.

**Comment by Rastuasi** [comment]
Devs should still look at the update process however, as to not have updates break existing configurations.

**Comment by Rastuasi** [comment]
Issue is back with latest version, only this time a download and refreshed  configuration folder did not help. With one main difference, this time X-Plane 12, in the network menu is declaring that one software on 127.0.0.1 is denying its data. As Smartcars and the sim are the only things running, SC3 is not accepting the data from the sim.

TLDR: After original post, a fresh download fixed it, I did several flights over a week. Went on a break of flying for a few days, came back to the client not working again and a fresh download did not fix it. It refuses to connect to the UDP of the simulator.

**Comment by Rastuasi** [comment]
[error-2024-07-26.log](https://github.com/user-attachments/files/16395557/error-2024-07-26.log)
[NoConnect_combined-2024-07-26.log](https://github.com/user-attachments/files/16395558/NoConnect_combined-2024-07-26.log)

Attached new test logs with yet another full wipe of SmartCARS 3 and install to try to connect. X-Plane is on the same machine, all stock connection settings outside changing fuel to US stuff and dark mode.

**Comment by turbofandude** [comment]
It seems the update process is messy on Linux. Thank you for following up and letting us know!

**Comment by Thien42** [comment]
Sorry for the delayed answer, it took us a while to get the latest version up and running.

Please download the latest version and let us know if you are still having issues.

**Comment by Rastuasi** [comment]
@Thien42  did a fresh download, completely broke everything. It shows sim as connected for 5-10 second then stops and ends up never showing my flight at the proper location. It also appears my entire VA (Walker Air) is facing the same issue.
